### PR TITLE
Add support for RFC6638 and RFC7953

### LINKED ISSUE #11: Implement RFC6638
- [ ] complete implementation for `ScheduleInbox`
- [ ] complete implementation for `ScheduleOutbox`

Includes `FreeBusy` support, scheduling, etc. 

https://tools.ietf.org/html/rfc6638

### LINKED ISSUE #12: Implement RFC7953
- [ ] allow setting of availability hours

https://tools.ietf.org/html/rfc7953

### PR DESCRIPTION
**Pull request by georgehrke**
Fixes #11
Fixes #12

### CODE REVIEW

File: src/propset/scheduleInboxPropSet.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import * as NS from '../utility/namespaceUtility.js';
```
Comment (tcitworld) [inline src/propset/scheduleInboxPropSet.js]: You may just import `IETF_CALDAV`

File: src/models/scheduleOutbox.js [hunk 2]
```diff
@@ -22,7 +22,44 @@
  */
 
 import { DavCollection } from './davCollection.js';
+import * as NS from '../utility/namespaceUtility.js';
```
Comment (tcitworld) [inline src/models/scheduleOutbox.js]: You may just import `resolve`

File: src/models/scheduleInbox.js [hunk 3]
```diff
@@ -21,8 +21,31 @@
  *
  */
 
-import { DavCollection } from './davCollection.js';
+import { Calendar } from './calendar.js';
+import * as NS from '../utility/namespaceUtility.js';
```
Comment (tcitworld) [inline src/models/scheduleInbox.js]: You may just import IETF_CALDAV

File: src/propset/scheduleInboxPropSet.js (hunk 1 above)
Comment (georgehrke) [reply src/propset/scheduleInboxPropSet.js]: Valid point, but we are using `import * as NS from '../utility/namespaceUtility.js';` everywhere in this lib right now. I think we should refactor it altogether at some point.

There is some bigger maintenance work needed in this lib anyway:
- improved imports
- use standard Nextcloud eslint style
- ...

**Pull request merged**